docs/APIBaselines.md

```diff
@@ -6,8 +6,24 @@ This document contains information regarding API baseline files and how to work
 
 When creating a new implementation (i.e. src) project, it's necessary to manually add API baseline files since API baseline are enabled by default. If the project is a non-shipping or test only project, add `<AddPublicApiAnalyzers>false</AddPublicApiAnalyzers>` to the project to disable these checks. To add baseline files to the project:
 
-1. Copy eng\PublicAPI.empty.txt to the project directory and rename it to PublicAPI.Shipped.txt
-1. Copy eng\PublicAPI.empty.txt to the project directory and rename it to PublicAPI.Unshipped.txt
+1. `cp .\eng\PublicAPI.empty.txt {new folder}\PublicAPI.Shipped.txt`
+1. `cp .\eng\PublicAPI.empty.txt {new folder}\PublicAPI.Unshipped.txt`
+1. Update AspNetCore.sln and relevant `*.slnf` file to include the new project
+1. `{directory containing relevant *.slnf}\startvs.cmd`
+1. F6 # or whatever your favourite build gesture is
+1. Click on a RS0016 (or whatever) error
+1. Right click in editor on underscored symbol or go straight to the “quick fix” icon to its left. Control-. also works.
+1. Choose “Add Blah to public API” / “Fix all occurrences in … Solution”
+1. Click Apply
+1. F6 # again to see if the fixer missed anything or if other RS00xx errors show up (not uncommon)
```

Comment:
The formatting on this comment didn't work

Reply:
Same above on line 13

Reply:
Not sure what the intended formatting was here. But I'll italicize